models/general/knowledge/en/description.txt

```diff
@@ -0,0 +1,25 @@
+!image:./images/description.png
+!example: Get started
+!description: When user queries with "Get started" during conversation with SUSI messenger bots
+Get started
+I am a personal assistant built by open source community FOSSASIA.
+
+!example: Start chatting
+!description: When user clicks "Start chatting" button in SUSI messenger bots
+Start chatting
+Ask me anything like temperature at a place or distance between two countries or any other thing which you would like to ask.
+
+!example: Follow back success
+!description: When user follows the SUSI A.I. account on Twitter and the SUSI account successfully follows back the user.
+Follow back success
+Thank you for following me. Tweet by tagging my name with your query, to get it answered. I followed you back so that you can also personally message me.
+
+!example: Follow back failed
+!description: When user follows the SUSI A.I. account on Twitter but the SUSI account couldn't follow back the user.
+Follow back failed
+Thank you for following me. Tweet by tagging my name with your query, to get it answered.
+
+!example: Welcome
+!description: When user tries chatting with SUSI A.I. for the first time.
+Welcome
+Welcome to SUSI A.I.! Please select an option:
```

Comment:
@Manveer695 when this skill will get merged and I will type Welcome in SUSI webchat. It will say
Welcome to SUSI A.I.! Please select an option:

Twitter bot support buttons that you have shown after thin but other clients don't so I think do not do it like this. Keep your answers general so that other clients can handle them also.